api/src/types/system/process.d.ts

```diff
@@ -14,5 +14,6 @@ declare namespace NodeJS {
     logger: import('@sasjs/utils/logger').Logger
     runTimes: import('../../utils').RunTimeType[]
     secrets: import('../../model/Configuration').ConfigurationType
+    dbInstance: import('mongoose').Mongoose
```

Comment:
I'm possibly out of my depth here, but should this be optional?  eg for MODE=desktop